### PR TITLE
Fix orientation of bad-belzig.geojson

### PR DESCRIPTION
**Pull request by sebkur**

### CODE REVIEW

**Comment by sebkur** [comment]
Tested this locally and works as expected now.

Used https://github.com/bjornharrtell/jts2geojson for reading this into JTS, then used `Geometry#normalize()` to fix the orientation and wrote back to GeoJSON using the jts2geojson. It uses a different property order for serialization, not sure if you care about that.

File: public/data/bad-belzig.geojson [hunk 1]
```diff
@@ -1 +1 @@
...
...00607],[12.5909521,52.1082873],[12.5871489,52.1080688],[12.5815248,52.1084786],[12.5766201,52.1092068],[12.5701881,52.1106452],[12.5644804,52.1121343],[12.5621844,52.1123451],[12.5593295,52.1118852],[12.5601674,52.1158766],[12.5609184,52.1181244],[12.5610794,52.1197582],[12.5602318,52.1227475],[12.5596095,52.1239806],[12.5615085,52.1263769],[12.5583006,52.1270224],[12.553537,52.1285505],[12.55233,52.1294047],[12.5505705,52.1320918],[12.5509191,52.1332529],[12.5498892,52.1369802],[12.5514556,52.1377177],[12.5512196,52.1385474],[12.54946,52.1404307],[12.5472928,52.1432883],[12.5469066,52.145461],[12.5472928,52.1464617],[12.5432373,52.1474361],[12.541113,52.147739],[12.5407697,52.1492155],[12.5414134,52.1509666],[12.5439454,52.1543107],[12.5452544,52.158181],[12.5457264,52.1582205],[12.5480439,52.1634463],[12.5442673,52.1641176],[12.5448252,52.1679344],[12.5483614,52.1692077],[12.5507475,52.1710403],[12.5513483,52.1740934],[12.5522453,52.1768666],[12.5461084,52.1809587],[12.5421859,52.1830671],[12.5395037,52.183317],[12.5395209,52.1849846],[12.5341779,52.1864975],[12.529336,52.1892232],[12.5261592,52.190763],[12.5246936,52.1922112]]]},"properties":{"id":-540424,"name":"Bad Belzig"}}]}
```
Comment (fnogatz) [inline public/data/bad-belzig.geojson:1]: Could you please add a newline at the very end of the file? :)

**Review by fnogatz — CHANGES_REQUESTED** [review]
Thank you very much, @sebkur! Bad Belzig now works as expected again.

File: public/data/bad-belzig.geojson (hunk 1 above)
Comment (sebkur) [reply public/data/bad-belzig.geojson]: sure, done

**Comment by fnogatz** [comment]
Thank you, @sebkur, for the requested change!

> It uses a different property order for serialization, not sure if you care about that.

Currently, we don't perform any normalization on the GeoJSON data (even the missing newline at the very end won't trigger the formatting checker on `npm test`).

btw, there's no need to force-push. I would have just squashed your commits when merging anyway :innocent:

**Comment by fnogatz** [comment]
Merged and online: https://click-that-hood.com/?bad-belzig :partying_face:
Thank you again, @sebkur!

**Pull request merged**